examples/diffuser/helpers.py

```diff
@@ -0,0 +1,267 @@
+import os
```

Comment:
I would guess just put these in the training file to minimize how many files things are across. Also huge points for getting rid of as many functions as you can (I did some of that in colab).